Unit-01/02-flask-routing/app.py

```diff
@@ -0,0 +1,24 @@
+from flask import Flask
+
+app= Flask(__name__)
+
+@app.route("/math/<operation>/<num1>/<num2>")
```

Comment:
instead of calling `int()` everywhere below, you can specify it in the route definition like so:

```python
@app.route("/math/<operation>/<int:num1>/<int:num2>")
```